src/reference/common_errors.md

```diff
@@ -80,6 +80,38 @@ Time synchronization instructions for Linux can be found [here](https://unix.sta
 1. Once the installation is complete, the NTP service should start automatically.
 1. Check its status by using this command `sudo systemctl status ntp`.
 
+## Error: Unexpected error when rendering on the frontend.
+
+**If you encounter this error, you are running Caido as root with `--no-renderer-sandbox`. In the [logs](/reference/configuration/data_location.md), you will see a similar message to the following:**
+
+```
+Rendering error: LaunchIo(Custom { kind: UnexpectedEof, error: "unexpected end of stream" }, BrowserStderr("[0101/110718.156035:ERROR:zygote_host_impl_linux.cc(90)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.\n"))
+```
+
+### Resolution:
+
+Don't run Caido as root and remove the flag `--no-renderer-sandbox` when launching Caido.
```

Comment:
Option 1: Don't run Caido as root
Option 2 (if you run caido as root):  Remove the flag `--no-renderer-sandbox` when launching Caido